week1/learnyounode/.vscode/extensions.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Make sure the `.vscode` files are not commited. You should add `.vscode` to your `.gitignore` file.
Since these are already commited you should remove them from git using `git rm .vscode` first, the commit, the add it to `gitignore` file